### PR TITLE
Add  queryset.exists()

### PR DESCRIPTION
**Pull request by long2ice**
## Description

Add  ``queryset.exists()``,``Model.exists()`` method
## Motivation and Context

resolve https://github.com/tortoise/tortoise-orm/issues/385
## How Has This Been Tested?

add ``test_exists()``
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tortoise/queryset.py [hunk 1]
```diff
@@ -536,6 +536,18 @@ def count(self) -> "CountQuery":
             offset=self._offset,
         )
 
+    def exists(self) -> "ExistsQuery":
```
Comment (grigi) [inline tortoise/queryset.py:539]: Could you add `, *args: Q, **kwargs: Any`
and then call `queryset = self.filter(*args, **kwargs)` and use that queryset instead?

That should allow for a shorter form of `await Desks.exists(owned_by=jon)` whilst still allowing the longer form that you implemented to continue working.

**Comment by grigi** [comment]
And could you also implement an `exists` on Model that just returns `Queryset(cls).exists(*args, **kwargs)` so we can support the short form as well?

Or, leave the exists in the queryset as you did it, but make this do .filter().exists() ?
That would be fine too.

File: tortoise/queryset.py (hunk 1 above)
Comment (long2ice) [reply tortoise/queryset.py:539]: I think do filtering in ``exists`` is not comfortable,it should like other methods like ``count()``,``first()`` etc,right?

**Comment by long2ice** [comment]
> And could you also implement an `exists` on Model that just returns `Queryset(cls).exists(*args, **kwargs)` so we can support the short form as well?
> 
> Or, leave the exists in the queryset as you did it, but make this do .filter().exists() ?
> That would be fine too.

This is OK

**Pull request merged**